jwMaxwell/Spike-2
# Logging with Winston

**Related bugs/feature requests**


+ Resolves #19 

**Describe the changes**

Uses the winston library to better control logging.

**Expected behavior**

Messages at various levels will be logged to files, the console, and discord.

**Screenshots**


**New npm dependencies**


- "@typicalninja21/discord-winston": "^1.0.1"
- "winston": "^3.3.3"
- "winston-daily-rotate-file": "^4.5.5"

**Testing Instructions**


- Run Simone at the `silly` log level
- Verify that everything is logging to the right spot at a reasonable level

**Additional context**

- [Wiki Article explaining the philosophy behind level decisions](https://github.com/jwMaxwell/Spike-2/wiki/SpikeKit:-Logging)
